pkg/clockcache/cache.go

```diff
@@ -16,6 +16,12 @@ type Cache struct {
 	elements map[interface{}]*element
 	storage  []element
 
+	//Information elements protected by elementsLock
```

Comment:
Go idiom is to group variables that are being protected by a lock together. Makes the code easier to read. 

I suggest removing the blank line above this one and cleaning up the comments.